src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -76,18 +75,19 @@ def _complex_params(command_group):
 
         with self.argument_context('{} server create'.format(command_group)) as c:
             c.argument('server_name', options_list=['--name', '-n'], arg_type=server_name_setter_arg_type)
-            c.argument('sku_name', options_list=['--sku-name'], required=True, help='The name of the sku. Follows the convention {pricing tier}_{compute generation}_{vCores} in shorthand. Examples: B_Gen5_1, GP_Gen5_4, MO_Gen5_16. ')
-            c.argument('administrator_login', required=True, arg_group='Authentication')
-            c.argument('administrator_login_password', required=True, arg_group='Authentication')
+            c.argument('sku_name', default='GP_Gen5_2', options_list=['--sku-name'],
+                       help='The name of the sku. Follows the convention {pricing tier}_{compute generation}_{vCores} in shorthand. Examples: B_Gen5_1, GP_Gen5_4, MO_Gen5_16. ')
+            c.argument('administrator_login', default=generate_username(), arg_group='Authentication')
+            c.argument('administrator_login_password', arg_group='Authentication')
 
             c.argument('backup_retention', type=int, options_list=['--backup-retention'], help='The number of days a backup is retained. Range of 7 to 35 days. Default is 7 days.', validator=retention_validator)
             c.argument('geo_redundant_backup', arg_type=get_enum_type(['Enabled', 'Disabled']), options_list=['--geo-redundant-backup'], help='Enable or disable geo-redundant backups. Default value is Disabled. Not supported in Basic pricing tier.')
-            c.argument('storage_mb', options_list=['--storage-size'], type=int, help='The storage capacity of the server (unit is megabytes). Minimum 5120 and increases in 1024 increments. Default is 51200.')
+            c.argument('storage_mb', default=51200, options_list=['--storage-size'], type=int, help='The storage capacity of the server (unit is megabytes). Minimum 5120 and increases in 1024 increments. Default is 51200.')
```

Comment:
May I know the change in client side? is it to align with server side as shown in description?

Reply:
Yes, for single servers, we are changing the default to 50 GB.